docs/COMPATIBILITY.md

```
@@ -14,150 +14,137 @@
+		<td>4.1.1 or higher needed for PHP 8.1+</td>
+	</tr>
+	<tr>
+		<th><a href="https://www.semantic-mediawiki.org/wiki/Semantic_MediaWiki_4.0.0">4.0.x</a></th>
+		<td>Obsolete release</td>
+		<td>2022-01-18</td>
+		<td>2022-07-21</td>
+		<td><strong>7.3</strong> - 8.0</td>
+		<td><strong>1.35</strong> - 1.37</td>
+		<td>Not compatible with PHP 8.1+ or MW 1.39</td>
+	</tr>
+	<tr>
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)